src/components/behandlerdialog/skrivtilbehandler/SkrivTilBehandlerSkjema.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import { SkjemaFeiloppsummering } from "@/components/SkjemaFeiloppsummering";
+import { FlexRow } from "@/components/Layout";
+import { Button } from "@navikt/ds-react";
+import { Form } from "react-final-form";
+import { VelgBehandler } from "@/components/behandlerdialog/skrivtilbehandler/VelgBehandler";
+import { BehandlerDTO } from "@/data/behandler/BehandlerDTO";
+import styled from "styled-components";
+import {
+  MeldingTekstfelt,
+  meldingTekstField,
+} from "@/components/behandlerdialog/skrivtilbehandler/MeldingTekstfelt";
+import {
+  behandlerRefValidationErrors,
+  validerSkjemaTekster,
+} from "@/utils/valideringUtils";
+import { useFeilUtbedret } from "@/hooks/useFeilUtbedret";
+
+const texts = {
+  knappTekst: "Send til behandler",
+};
+
+const validationTexts = {
+  missingMeldingTekst: "Innholdet i meldingen er tomt", // TODO: se nærmere på teksten
+};
```

Comment:
Trenger vi eget text-objekt for dette? Kan det ligge i vanlig `texts`, evt. `texts = {tekst1: "tekst1", validation: {mintekst: "asd"}}`

Reply:
Ja, kan slå det sammen 👍🏼